include/vigra/mathutil.hxx

```diff
@@ -164,6 +164,17 @@ VIGRA_DEFINE_MISSING_ABS(signed long long)
 using std::isinf;
 using std::isnan;
 
+#ifdef VIGRA_NO_INT_ISNAN
```

Comment:
Understood, but unfortunately he did not report the candidate matches to investigate this further. @hmeine, can you add this information?

I still don't like the fact that we need cmake to find out if `VIGRA_NO_INT_ISNAN` needs to be defined, but I cannot suggest a better way because I have no access to a platform where it is required.